assets/src/wizards/subscriptions/index.js

```diff
@@ -31,43 +34,99 @@ class SubscriptionsWizard extends Component {
 	constructor() {
 		super( ...arguments );
 		this.state = {
+			currentTask: 0,
 			inputTextValue1: "Input value",
 			inputTextValue2: "",
 			image: null,
+			completedTasks: [ 2 ],
 			selectValue1: "2nd",
 			selectValue2: "",
 		}
 	}
 
 	/**
-	 * Render the example stub.
+	 * componentDidMount.
+	 */
+	componentDidMount() {
+		this.setState( { currentTask: this.getNextTask( 0, this.state.completedTasks) } )
+	}
+
+	/**
+	 * Starting from the startingIndex, determine the next task to go to.
+	 *
+	 * @param int startingIndex
+	 * @param Array completedTasks
+	 * @returns int {number}
+	 */
+	getNextTask = ( startingIndex, completedTasks ) => {
+		let nextTaskIndex = 0;
+		while ( completedTasks.indexOf( nextTaskIndex ) !== -1 ) {
+			nextTaskIndex++;
+		}
+
+		return nextTaskIndex;
+	}
```

Comment:
- here added a handler when the task is complete (besides updating the `currentTask`, now also adding it into the `completedTasks` Array):

```suggestion
	}

	/**
	 * Upon completing the task, add its index to this.state.completedTasks, and increase this.state.currentTask.
	 *
	 * @param int taskIndex
	 */
	completeTask = ( taskIndex ) => {
		let completedTasks = [ ...this.state.completedTasks ];
		if ( completedTasks.indexOf( taskIndex ) === -1 ) {
			completedTasks.push( taskIndex );
			completedTasks.sort();

			this.setState( { completedTasks } );
		}

		this.setState( { currentTask: this.getNextTask( 0, completedTasks ) } );
	}
```

Reply:
**This comment is a dupe, please ignore.**